plugin-lifecycle/README.md

```diff
@@ -0,0 +1,41 @@
+## Mapbox Maps Lifecycle Plugin for Android
+
+### Overview
+
+The Mapbox Maps Lifecycle Plugin for Android is an public library for handling Lifecycle events of the Mapbox SDK MapView. By default, the Lifecycle Plugin will automatically hook into the lifecycle events(onStart/onStop/onDestroy) of the host Activity or Fragment, so that developers doesn't need to manually invoke the corresponding lifecycle methods of MapView themselves.
```

Comment:
```suggestion
The Mapbox Maps Lifecycle Plugin for Android is an public library for handling Lifecycle events of the Mapbox SDK MapView. By default, the Lifecycle Plugin will automatically hook into the lifecycle events(onStart/onStop/onDestroy) of the host Activity or Fragment, so that developers don't need to invoke the corresponding lifecycle methods of MapView manually themselves.
```